packages/adblocker/src/engine/engine.ts

```diff
@@ -28,6 +28,7 @@ import Request from '../request';
 import Resources from '../resources';
 import CosmeticFilterBucket from './bucket/cosmetic';
 import NetworkFilterBucket from './bucket/network';
+import { Metadata, ITrackerLookupResult } from './metadata';
 
 export const ENGINE_VERSION = 572;
```

Comment:
Is adding a bool a breaking change? Should it require to increase the ENGINE_VERSION?